src/Mutator/ZeroIteration/For_.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutator\ZeroIteration;
+
+use Infection\Mutator\Mutator;
+use PhpParser\Node;
+
+class For_ extends Mutator
```

Comment:
Do we need the trailing `_`?

Reply:
For is a reserved word by PHP, so it can't be used as a class name.

Reply:
`ForCondition`?

Reply:
`For_` is in line with all our other mutators

Reply:
Fair then :/